python/processors/aptos_ambassador_token/processor.py

```diff
@@ -0,0 +1,83 @@
+import argparse
+import grpc
+import ast
+import json
+
+from aptos.indexer.v1 import raw_data_pb2_grpc
+from aptos.indexer.v1 import raw_data_pb2
+from aptos.transaction.testing1.v1 import transaction_pb2
+from utils.config import Config
+
+parser = argparse.ArgumentParser()
+parser.add_argument(
+    "-c", "--config", help="Path to config file", required=True)
+args = parser.parse_args()
+config = Config.from_yaml_file(args.config)
+starting_version = 0
```

Comment:
You can get the starting version from the Config object now. [Code pointer](https://github.com/aptos-labs/aptos-indexer-processors/blob/main/python/utils/config.py#L38)

Reply:
I couldn't use it because this example doesn't need to establish the DB connection which this line of code does: https://github.com/aptos-labs/aptos-indexer-processors/blob/main/python/utils/config.py#LL39C28-L39C28